fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/mixin/resource/loader/client/KeyedResourceReloadListenerClientMixin.java

```diff
@@ -36,11 +38,13 @@
 
 @Mixin({
 		/* public */
-		SoundLoader.class, BakedModelManager.class, LanguageManager.class, TextureManager.class,
+		SoundManager.class, BakedModelManager.class, LanguageManager.class, TextureManager.class,
 		/* private */
 		WorldRenderer.class, BlockRenderManager.class, ItemRenderer.class
```

Comment:
You added keys for some of these?

Reply:
All of these either had keys, or do now.

Reply:
Well my assumption was the private comment meant there was no key, so the mixins should be moved up?

Reply:
Oh, I see what you mean. IDK why they were private before tbh. I might undo that for now, as this is supposed to be a bug fix PR.